src/components/backdrop/backdrop.js

```diff
@@ -66,6 +71,13 @@ angular
         }
       });
 
+      function resize() {
```

Comment:
* The width is always 100%. 
* I'm not sure about margins, I just moved the logic to a separate function and attached it to the viewport resize. I don't see a situation where the backdrop might have a margin, though.